analysis.go

```diff
@@ -51,11 +60,15 @@ type DeploymentConfigs struct {
 	} `json:"items"`
 }
 
-type ContainerWaiting struct {
+// ContainerStateWaiting is a possible status that a pods containers can be in.
+// This structure is cloned from:
+// https://github.com/openshift/origin/blob/570a4a3d0eb4c56d1b7cb1eef31ce9adcc0ec95a/vendor/k8s.io/kubernetes/pkg/api/types.go#L1052
```

Comment:
Hmm, I know I suggested this :)

But I meant it more generally... why would we add a note with such a link to one of the types and leave all the others without?

We could tell where they come from in a common comment preceding all definitions, and the other suggestion was to copy doc comment + type definition from origin, then remove the struct fields we don't care about.